common/types/explorer.ts

```diff
@@ -380,7 +380,7 @@ export interface VisMeta {
 }
 
 export interface VisualizationState {
-  query: Query;
+  queryState: Query;
```

Comment:
any reason why this was changed?

Reply:
linting fix, there's a global 'query' within the context where inner query will shadow the global one.